composer.json

```diff
@@ -44,6 +44,6 @@
     "typo3-ter/pxa-form-enhancement": "~3.0.0",
     "typo3-ter/pxa-cookie-bar": "~1.0.7",
     "svewap/realurl-404-multilingual": "~1.0.9",
-    "typo3-ter/frontend-editing": "~1.2.2"
+    "typo3-ter/frontend-editing": "~1.2.3"
```

Comment:
Why is this needed? The given version contraint already include version "1.2.3".
Or there is a reason, why the extension "theme_t3kit" not working with version 1.2.2 of "frontend_editing"?

See https://getcomposer.org/doc/articles/versions.md

Reply:
@websi I just want to make sure that we always will run the version 1.2.3 as it contains some nifty improvements.

Reply:
This is ensured for both variants.
You should use `composer update t3kit/theme-t3kit --with-dependencies`